ouroboros-consensus/test-consensus/Test/Dynamic/Network.hs

```diff
@@ -518,23 +559,36 @@ getTestOutput ::
         , NodeConfig (BlockProtocol blk)
         , NodeKernel m NodeId blk
         , NodeInfo blk (StrictTVar m MockFS)
+        , m [(SlotNo, BlockNo)]
+        , m [TraceForgeEvent blk]
+        , m [Point blk]
```

Comment:
Hmm, I think need a record for these. Maybe even one bundling all seven arguments?

Reply:
I combined these three with `NodeInfo`. Are you OK with the so-called "higher-kinded data" (eg as used in `NodeTracers`)?

Reply:
Yup, HKD is fine for me, the renaming is also good.